tests/nfs/run.pm

```diff
@@ -32,15 +32,24 @@ sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
-    script_run('cd ~/pynfs');
-    server_test_all;
+    if (get_var("PYNFS")) {
+        script_run('cd ~/pynfs');
+        pynfs_server_test_all;
+    }
+    elsif (get_var("CTHON04")) {
+        scritp_run('cd ~/cthon04');
```

Comment:
typo: scritp_run => script_run :)

Reply:
I must need a new glasses lol

Reply:
And it works, I updated it in verify run. http://10.67.133.102/tests/327